weaver/wps_restapi/processes/processes.py

```diff
@@ -201,6 +202,32 @@ def execute_process(self, job_id, url, headers=None, notification_email=None):
     return job.status
 
 
+def set_wps_language(wps, accept_language):
+    # type: (WebProcessingService, str) -> None
+    """Set the `language` property on the `WebProcessingService` object.
+
+    Given the `accept_language` header value, match the best language
+    to the supported languages.
+
+    By default, and if no match is found, the `language` property is None.
+
+    https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language
+    (q-factor weighting is ignored, only order is considered)
+    """
```

Comment:
Adjust docstrings to point to actual property/class using RST convention.

Reply:
Done